platform/darwin/scripts/generate-style-code.js

```diff
@@ -77,17 +83,33 @@ global.testHelperMessage = function (property, layerType, isFunction) {
     }
 };
 
-global.propertyDoc = function (property, layerType) {
-    let doc = property.doc.replace(/`(.+?)`/g, function (m, symbol, offset, str) {
+global.propertyDoc = function (propertyName, property, layerType) {
+    // Match references to other property names & values. 
+    // Requires the format 'When `foo` is set to `bar`,'.
+    let doc = property.doc.replace(/When `(.+?)` is set to `(.+?)`,/g, function (m, peerPropertyName, propertyValue, offset, str) {
```

Comment:
I tried without the `When` and comma, but ran into issues that crossed sentence boundaries. I can try to refine it further.